controlplane/pkg/apis/registry/registry.proto

```diff
@@ -92,4 +98,5 @@ message SubnetExtendingResponse {
 service ClusterInfo {
```

Comment:
We are moving ClusterInfo stuff out of registry.proto...

Reply:
Please do migrate the ClusterInfo stuff out of the registry.proto as discussed in #1243

Reply:
I suggest to make this migration within another pr.